libcontainer/standard_init_linux.go

```diff
@@ -21,6 +20,7 @@ import (
 type linuxStandardInit struct {
 	pipe      io.ReadWriteCloser
 	parentPid int
+	rootfd    int
```

Comment:
nit: Can we call this `statedirfd`? `rootfd` seems to imply that it is a fd to the rootfs.